### PR TITLE
Change regression current class label

### PR DESCRIPTION
**Pull request by zhb000**
Change regression current class label

### CODE REVIEW

File: libs/counterfactuals/src/lib/CounterfactualChart.tsx [hunk 1]
```diff
@@ -379,13 +379,14 @@ export class CounterfactualChart extends React.PureComponent<
   }
 
   private getCurrentClass(): string {
```
Comment (imatiach-msft) [inline libs/counterfactuals/src/lib/CounterfactualChart.tsx:381]: minor nit: maybe getCurrentClass should be renamed to something else, like getCurrentLabel, since in the regression case classes don't apply like in the classification scenario?

File: libs/counterfactuals/src/lib/CounterfactualChart.tsx [hunk 2]
```diff
@@ -379,13 +379,14 @@ export class CounterfactualChart extends React.PureComponent<
...
-    if (this.props.data.desired_class) {
-      return this.props.data.desired_class;
-    }
-    if (this.props.data.desired_range) {
-      return this.props.data.desired_range.join("->");
+    if (this.context.dataset.task_type === "regression") {
+      return (
+        (this.props.data.desired_range &&
```
Comment (imatiach-msft) [inline libs/counterfactuals/src/lib/CounterfactualChart.tsx:384]: is it possible to add some kind of tooltip as well to explain what this means in the what if tool?  that can be a separate PR and maybe might need some PM input

Comment (zhb000) [reply libs/counterfactuals/src/lib/CounterfactualChart.tsx:384]: It could cause inconsistent if just add tooltip for regression cases, 
Also, the prefix "Current class" maybe need to rephrased, as in regression it's a range instead of specific class?
@minthigpen any idea?

**Review by imatiach-msft — APPROVED** [review]
